src/main/webapp/main/LandingPage.js

```diff
@@ -66,19 +66,22 @@ const LandingPage = hh(class LandingPage extends Component{
     };
   }
 
-  componentDidMount() {
+  componentDidMount = async () => {
     this._isMounted = true;
-    this.getProjectsList();
-    this.getTaskList();
-    this.hasSession();
-  }
+    this.props.showSpinner();
+    await this.getProjectsList();
+    await this.getTaskList();
+    await this.hasSession();
```

Comment:
Is there a way to await outside of componentDidMount? I say this because it looks like these three awaits turn them into serial calls instead of parallel ones.

Reply:
something like Promise.all or axios.all ? This patter is repeated in several components so we may find a general approach and fix all cases.

Reply:
Yes, exactly. Or, a separate async function that is called from componentDidMount and *that* function is `await`-ed from inside componentDidMount

Reply:
We've just updated this, please check our approach, and let us know what do you think.